packages/aircrack-ng/build.sh

```diff
@@ -1,21 +1,22 @@
-TERMUX_PKG_HOMEPAGE=https://www.aircrack-ng.org/
+TERMUX_PKG_HOMEPAGE="https://www.aircrack-ng.org/"
 TERMUX_PKG_DESCRIPTION="WiFi security auditing tools suite"
 TERMUX_PKG_LICENSE="GPL-2.0"
-TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:1.6
-TERMUX_PKG_REVISION=3
-TERMUX_PKG_SRCURL=https://github.com/aircrack-ng/aircrack-ng/archive/${TERMUX_PKG_VERSION:2}.tar.gz
-TERMUX_PKG_SHA256=c9e7894ba30f5e45b8a20ec43b4599225ac739a795a5bdb98e3f1bbb854d54f7
-TERMUX_PKG_DEPENDS="libc++, libnl, openssl, libpcap, pciutils, ethtool"
+TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
+_COMMIT=3f9be67b1eaf84057d7f9830883b055817683915
+TERMUX_PKG_VERSION="2021.12.19-${_COMMIT:0:8}"
```

Comment:
The epoch, `1:`, need to be kept. 2021 will probably be considered higher than 1.6, but just to be sure I suggest to change to `2:2021..`